src/test/java/io/moderne/organizations/OrganizationDataFetcherTest.java

```diff
@@ -19,31 +19,31 @@ public class OrganizationDataFetcherTest {
     void organizationsStructure() {
         StepVerifier
                 .create(organizationDataFetcher.allOrganizations())
```

Comment:
See this comment for why this [test](https://github.com/moderneinc/moderne-organizations/pull/98/files#r1752865054) has changed

Reply:
I'm wondering if there are any implications with this order change. I doubt it does, but we should make sure this does not change the order in the UI.

Reply:
after reading the other comment (apparently on a different commit, so I missed it) I agree. If this works in multi-tenant it will work here too.